ui/app/models/pki/crl.js

```diff
@@ -4,15 +4,67 @@
  */
 
 import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
+@withFormFields(['expiry', 'autoRebuildGracePeriod', 'deltaRebuildInterval', 'ocspExpiry'])
 export default class PkiCrlModel extends Model {
   // This model uses the backend value as the model ID
-  get useOpenAPI() {
-    return true;
-  }
 
-  @attr('string') expiry;
   @attr('boolean') autoRebuild;
-  @attr('string') ocspExpiry;
+  @attr('string', {
```

Comment:
I considered making these `ttls` and creating a new transform, but I didn't see the immediate value in doing that. But might be a nice follow on, to have an attribute option that chose which key of the ttl object (i.e. `goSafeTimeString`) the att corresponds to